lectures/tools_and_techniques/scalar_dynam.md

```diff
@@ -322,35 +306,155 @@ The initial condition is $k_0 = 0.25$.
 
 ```{code-cell} julia
 k0 = 0.25
-
-plot45(g, p.xmin, p.xmax, k0, num_arrows=5)
+plot45(k -> g(k; p), p.xmin, p.xmax, k0, num_arrows=5)
 ```
 
 We can plot the time series of capital corresponding to the figure above as
 follows:
 
 ```{code-cell} julia
-ts_plot(g, p.xmin, p.xmax, k0)
+ts_plot(k -> g(k; p), p.xmin, p.xmax, k0)
 ```
 
 Here's a somewhat longer view:
 
 ```{code-cell} julia
-ts_plot(g, p.xmin, p.xmax, k0, ts_length=20)
+ts_plot(k -> g(k; p), p.xmin, p.xmax, k0, ts_length=20)
 ```
 
 When capital stock is higher than the unique positive steady state, we see that
 it declines:
 
 ```{code-cell} julia
 k0 = 2.95
-
-plot45(g, p.xmin, p.xmax, k0, num_arrows=5)
+plot45(k -> g(k; p), p.xmin, p.xmax, k0, num_arrows=5)
 ```
 
 Here is the time series:
 
 ```{code-cell} julia
-ts_plot(g, p.xmin, p.xmax, k0)
+ts_plot(k -> g(k; p), p.xmin, p.xmax, k0)
+```
+
+### Complex Dynamics
+
+The Solow model is nonlinear but still generates very regular dynamics.
+
+One model that generates irregular dynamics is the **quadratic map**
+
+$$
+g(x) = 4 x (1 - x),
+\qquad x \in [0, 1]
+$$
+
+Let's have a look at the 45 degree diagram.
+
+```{code-cell} julia
+xmin, xmax = 0, 1
+g(k) = 4 * k * (1 - k)
+x0 = 0.3
+plot45(k -> g(k), xmin, xmax, 0, num_arrows=0)
```

Comment:
This should just be `plot45(g, xmin, xmax, 0, num_arrows=0)` 

No need for the closure since it is a univariate function.